### ISSUE TITLE
Add support for setting anti-aliasing style (and other styles)

### ISSUE THREAD

**Issue by bjz** [Gekkio/imgui-rs#30]
I would like to set the anti-alias mode on my UI to `false`:

```rust
pub struct ImGuiStyle {
    ...
    pub anti_aliased_lines: bool,
    pub anti_aliased_shapes: bool,
    ...
}
```

Would be nice if there was a way to do this in the API

**Comment by Gekkio** [comment]
It's already possible:

```rust
let style = imgui.style_mut();
style.anti_aliased_lines = false;
style.anti_aliased_shapes = false;
```

**Comment by bjz** [comment]
Hmm - doesn't seem to change the rendering of the UI... :/

**Comment by Gekkio** [comment]
That's weird...changing colours works, so the style changes are definitely applied:

```rust
style.colors[0] = (0.0, 1.0, 1.0, 1.0).into();
```

**Comment by Gekkio** [comment]
It's an alignment error...I'll fix it in a minute

**Comment by Gekkio** [comment]
Fixed in 31c2d9f5b7b9b2a81c672294fed79280b63602d7

**Comment by bjz** [comment]
That works!
